Code/CSharpLINQ/Cat.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpLINQ
+{
+    internal class Cat
+    {
+        public string Color { get; set; }
+
+
+        public void Task_09()
+        {
+            List<string> colors = new List<string> { "grey", "white", "black" };
+            List<Cat> cats = colors.Select(color => new Cat { Color = color }).ToList();
+
+            Console.WriteLine("The list of cats: ");
+
+            foreach (var cat in cats)
+            {
+                Console.WriteLine("Color is: " + cat.Color);
+            }
+        }
+        public void Task_10()
+        {
```

Comment:
In this task final list should contain list of property values :)
 Moreover action ".Select(cat => new Cat { Color = cat.Color })" is not required here as we already have a list of Cats

Reply:
done